ocfweb/tv/templates/tv/desktop_usage.html

```diff
@@ -0,0 +1,16 @@
+<div class="page-tv-hours">
+  <p>There are <span id="desktop_num">_</span> desktops available.</p>
+  <script>
+    $(document).ready(function() {
+      function update() {
+        $.getJSON("{% url 'desktop_usage' %}", function(res) {
+          document.getElementById("desktop_num").innerHTML =
+          (res.total_desktops - res.desktops_in_use.length);
+        });
+      }
+
+      update();
+      setInterval(update, 15000);
```

Comment:
Nice, I think this is a decent amount of time to give between requests. You could even go lower if you want, but I think this is good.

Reply:
I am thinking about updating it 1-5 seconds on the ocfside and 5-10 seconds on js. So people use this website would have a maximum of 15 second lag. If we go with 1s + 5s, would that be a burden/spam to our server? Imo fast update is pretty important for in-lab users waiting to get a desktop they cannot see.

Reply:
I think it should be fine with what you have right now, although we might want to make the SQL query a bit faster/easier maybe?